src/Mealz/MealBundle/Resources/config/routing.yml

```diff
@@ -124,6 +166,10 @@ MealzMealBundle_DishVariation_delete:
     path:  /dish/variation/{slug}/delete
     defaults: { _controller: App\Mealz\MealBundle\Controller\DishVariationController::deleteAction }
 
+MealzMealBundle_Dishes:
+  path: /dishes
+  defaults: { _controller: App\Mealz\MealBundle\Controller\TestController::renderIndex }
```

Comment:
Are we using this controller? If yes, then we should rename it to something understandable.